doc/rados/operations/health-checks.rst

```diff
@@ -801,6 +801,92 @@ Or, to disable this alert on a specific OSD, run the following command:
 
    ceph config set osd.123 bluestore_warn_on_spurious_read_errors false
 
+BLOCK_DEVICE_STALLED_READ_ALERT
+_______________________________
+
+There are some logs found in OSD logging triggerred by BlueStore which can 
+cause bad disk. This can lead to degradation of performance in the cluster 
+before identifying the bad disk as the cause.
+
+``read stalled read 0x29f40370000~100000 (buffered) since 63410177.290546s, timeout is 5.000000s``
+
+However, this is difficult to spot as there's no discernible warning (a
+health warning or info in ``ceph health detail`` for example). More observations
+can be found here: https://tracker.ceph.com/issues/62500
+
+As there can be false positive ``stalled read`` indications, a mechanism
+has been added for more reliability. If in last ``bdev_stalled_read_warn_lifetime``
+duration ``stalled read`` indications are found more than or equal to
```

Comment:
s/indications/instances/

Reply:
@sajibreadd , @anthonyeleven My proposal:
....
has been added for more reliability. Within ``bdev_stalled_read_warn_lifetime`` time window
we look to ``stalled read`` events. If there are at least ``bdev_stalled_read_warn_threshold``
in the window a warning appears in ``ceph health detail`` and is removed when condition clears.
Each BlueStore block device is supervised separately and warning are formatted accordingly.

Reply:
I can wordsmith this in a followup PR.